RecoTauTag/RecoTau/python/tools/runTauIdMVA.py

```diff
@@ -763,3 +717,11 @@ def processDeepProducer(self, producer_name, tauIDSources, workingPoints_):
                         self.cms.InputTag(producer_name, 'VS{}{}'.format(target, point)))
 
             setattr(getattr(self.process, producer_name), 'VS{}WP'.format(target), cuts)
+
+
+    def getDpfTauVersion(self, file_name):
+        version_search = re.search('201[125678]v([0-9]+)[\._]', file_name)
```

Comment:
It is quite clear, but please add a comment about expected pattern in a file name for those who might not understand regex syntax; something like this: "File name should contain a version label with data takig year (2011-2, 2015-8) and version number (vX), e.g. 2017v0."